src/ocm/components/HostsClusterDetailTab/HostsClusterDetailTabContent.tsx

```diff
@@ -23,20 +23,20 @@ export const HostsClusterDetailTabContent = ({
   isVisible,
 }: HostsClusterDetailTabProps) => {
   const [error, setError] = React.useState<ReactNode>();
-  const [day2Cluster, setDay2Cluster] = useStateSafely<Cluster | null | undefined>(undefined);
+  const [day2Cluster, setDay2Cluster] = useStateSafely<Cluster | null>(null);
```

Comment:
I think there was a reason why the distinction between `null` and `undefined` was made. Looks related to the comment about "ensure exclusive run". I would not change this unless we understand the impact.

Maybe @jgyselov knows?

Reply:
That's part of some legacy code, I've been working with Montse in order to eliminate it.
There was no actual difference between the day2Cluster when it was undefined and null. Therefore we managed to simplify its type.